modules/osx/init.zsh

```diff
@@ -19,3 +19,13 @@ alias cdf='cd "$(pfd)"'
 
 # Pushes directory to the current Finder directory.
 alias pushdf='pushd "$(pfd)"'
+
+# Locks screen.
+alias lock='/System/Library/CoreServices/Menu\ Extras/User.menu/Contents/Resources/CGSession -suspend'
```

Comment:
Seems like overkill, something like `pmset displaysleepnow` is nicer and isn't as slow to show the login prompt since it doesn't initiate fast user switching.

Reply:
Looks like the `pmset` approach is the only remaining one on Big Sur/Monterey... `User.menu` is gone, now that the Control Center has supplanted most Menu Extras.